microhapulator/data/template.html

```diff
@@ -220,12 +226,21 @@ <h2>Read Mapping</h2>
             </table>
             </div>
             <br />
-            <p>The following table shows the number of total reads mapped to each marker for each sample. Also shown is the number of off-target reads, determined by selecting the reads aligned to the marker, mapping them to the entire GRCh38 human reference sequence, and counting the reads that map preferentially to a locus outside the target marker locus.</p>
+
+            <p> The following donut plots show the proportions of on target, repetitive, off target, and cotaminant reads. On target reads are those reads that align preferentially to the marker locus when aligned to the full human reference genome. Repetitive reads align to a marker sequence but preferentially align to a different locus when aligned to the full reference. Off target reads do not align to any marker sequences but do align elsewhere in the human genome. Lastly, contaminant reads do not align anywhere in the human genome.</p>
+       
+            {% for plot in plots["donut"] %}
+            <img src="{{plot}}"  class="centerfull"/>
+            <br />
+            {% endfor %}
```

Comment:
My concerns about legibility, and request for one plot per line, was related to the font size of the wedge annotations. Now that we've abandoned those, I think we can go back to two plots per line.

```
{% for plot in plots["donut"] %}
<img src="{{plot}}" />
{% endfor %}
```